src/backend/api/meals.js

```diff
@@ -93,5 +138,19 @@ router
       }
     }
   });
-
+router.get("/:meal_id/reviews", async (req, res) => {
+  const id = req.params["meal_id"];
+  if (isNaN(id) || !id || id === "meal_id") {
```

Comment:
In my perception only: `if(isNaN(id)){ ... }` is enough.
Why?
- because if `id` is null or undefined the route is gonna match.
- the last check `id === 'meal_id'` is already covered by the first check `isNaN(id)`. This check `isNaN(id)` verifies that the `id` is not a string or character. So, no need to add  `id === 'meal_id'`